setup/updating.md

```diff
@@ -65,8 +65,54 @@ To update Coder, follow these steps:
    ```console
    helm upgrade --namespace coder --install --atomic --wait \
      --version 1.16.1 coder coder/coder --values current-values.yaml
+
+   ```
+
+1. If you're upgrading from v1.20 to v1.21,
+   [update your Helm chart](../guides/admin/helm-charts.md) via the steps below.
+   Refer to the `1.21`
+   [helm chart](https://github.com/cdr/enterprise-helm/blob/main/values.yaml)
+   when making such changes:
+
+   Rename all instances of `cemanager` to `coderd`.
+
+   Redirect all traffic from your custom ingress to `coderd`.
+
+   `coderd` now serves TLS certificates; to upgrade the NGINX ingress
+   controller, set `coderd.serviceNext` to `true`. (See our
+   [TLS certificates](../guides/tls-certificates/index.md) for more information
+   on how to configure Coder to issue and use certificates.)
+
+   > To manage SSH, go to **Manage** > **Providers**. Select the provider you
+   > want to modify, click the **vertical ellipses** to its right, and click
+   > **Edit**. Under **Features**, you can toggle **External Connect** on/off.
+   >
+   > To manage the Access URL, go to **Manage** > **Admin** >
+   > **Infrastructure**.
+
+1. Migrate the following ports (this may need to be done using your cloud
+   provider):
+
+   Move TCP port `8080` to `80`.
+
+   Move TCP port `8443` to `443`.
```

Comment:
Where do you migrate the ports? I'm a bit confused by this

Reply:
I believe this would need to be done at the Load Balancer level (if there is one). @mterhar please confirm or clarify.